charts/brigade-github-app/templates/_helpers.tpl

```diff
@@ -34,7 +34,17 @@ Return the appropriate apiVersion for a networking object.
 {{- define "networking.apiVersion" -}}
 {{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
 {{- print "extensions/v1beta1" -}}
-{{- else -}}
+{{- else if semverCompare "<1.19-0" .Capabilities.KubeVersion.GitVersion -}}
 {{- print "networking.k8s.io/v1beta1" -}}
+{{- else -}}
+{{- print "networking.k8s.io/v1" -}}
+{{- end -}}
```

Comment:
Did we pick up an extra `end` here?

Reply:
No. I think this is just misleading formatting.
Two `end`s is correct. One is needed for the `define`, the other for the `if`.

I think it is shown as addition because the last `end` in the file (line 50), isn't shown as addition.